src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -95,4 +103,44 @@ void parseMethodCalls() {
         assertEquals("test", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").getFirst()).get(1));
         assertEquals("fark", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(1)).get(1));
     }
+
+    private void assertUnsupportedOperation(int type, List<StringInt> parsedEntries, BibDatabaseContext context) {
+        LayoutEntry layoutEntry = new LayoutEntry(parsedEntries, type, null, null, null);
+        assertThrows(UnsupportedOperationException.class, () -> layoutEntry.doLayout(context, StandardCharsets.UTF_8));
+    }
+
+    private void assertLayoutResult(int type, String expected, List<StringInt> parsedEntries, BibDatabaseContext context) {
+        LayoutEntry layoutEntry = new LayoutEntry(parsedEntries, type, null, null, null);
+        assertEquals(expected, layoutEntry.doLayout(context, StandardCharsets.UTF_8));
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+            "2",
+            "3",
+            "4",
+            "6",
+            "7"
+    })
+    void testUnsupportedOperationTypes(int type) {
+        List<StringInt> parsedEntries = List.of(new StringInt("place_holder", 0),
+                new StringInt("testString", 0));
+        BibDatabaseContext bibDatabaseContext = new BibDatabaseContext(new BibDatabase(), new MetaData(), null);
+        assertUnsupportedOperation(type, parsedEntries, bibDatabaseContext);
```

Comment:
Inline the helper method - that helper method is used only here. No need to have a separeate method for two liens of code.